Makefile

```diff
@@ -273,24 +273,33 @@ test-nightly-server: build-nightly-server
 
 ### Check Rules: static checks, quality tools ###
 
-check: check-shell-scripts
+check: check-shell-scripts check-yaml
 .PHONY: check
-
 # rule requires shellcheck and find to run
 check-shell-scripts: $(filter $(ALT_BIN)%,$(SHELLCHECK))
 	$(SHELLCHECK) -P tests/ -eSC2181 -fgcc $$(find $(ROOT_DIR) -name "*.sh")
 .PHONY: check-shell-scripts
 
+
+check-yaml: $(filter $(ALT_BIN)%,$(YAMLLINT_CMD))
+	$(YAMLLINT_CMD) -c $(CURDIR)/.yamllint.yaml $(CURDIR)
+.PHONY: check-yaml
+
 # not included in check to not disrupt wip branches
 check-gitlint: $(filter $(ALT_BIN)%,$(GITLINT))
 	$(GITLINT) -C .gitlint --commits origin/master.. lint
 .PHONY: check-gitlint
 
 ### Misc. Rules ###
 
-clean:
-	$(RM) $(BUILDFILE_PREFIX)*
+clean: clean-buildfiles clean-altbin
 .PHONY: clean
+clean-buildfiles:
+	$(RM) $(BUILDFILE_PREFIX)*
+.PHONY:  clean-buildfiles
+clean-altbin:
+	$(RM) -r $(ALT_BIN)
+.PHONY: clean-altbin
```

Comment:
Previous occurrence of `clean-altbin` target still exists towards the end of Makefile. I think its better to have everything(clean related rules) towards the end of Makefile.

Reply:
fixed, thanks!